CHANGELOG.md

```diff
@@ -3,6 +3,58 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 3.0.0-beta.0 (2019-10-07)
+
+### Infrastructure:
+
+- feat(banner): Update CSS styles to match React ([#50](https://github.com/Workday/canvas-kit/pull/50))
+- feat(modal): Implement CSS Modal to match React implementation ([#185](https://github.com/Workday/canvas-kit/pull/185))
+- feat(core): Add border radius variables to CSS and React ([#204](https://github.com/Workday/canvas-kit/pull/204))
+- feat(button): Add blue css buttons ([#231](https://github.com/Workday/canvas-kit/pull/231))
+- feat(button): Promote React beta Button ([#191](https://github.com/Workday/canvas-kit/pull/191))
+- docs: Update component_status.md
```

Comment:
This and #240 are the only things in this list that are infrastructure. Everything else is related to components